cli/utils.ts

```diff
@@ -0,0 +1,40 @@
+import * as fs from "fs";
+import * as protos from "@dataform/protos";
+
+export enum WarehouseTypes {
+  BIGQUERY = "bigquery",
+  REDSHIFT = "redshift",
+  SNOWFLAKE = "snowflake"
+}
+
+export function readProfile(profilePath: string): protos.IProfile {
+  return fs.existsSync(profilePath) ? JSON.parse(fs.readFileSync(profilePath, "utf8")) : null;
+}
+
+export function validateProfile(profile: protos.IProfile): void {
+  // profile shouldn't be empty
+  if (!profile || !Object.keys(profile).length) {
+    throw new Error("Missing profile JSON file or file is empty.");
```

Comment:
Then here we can just say - Profile file is empty.

Reply:
(And in both these errors, let's say what the file path we are trying to read is)